### PR TITLE
vibrate

### PR DESCRIPTION
**Pull request by adrsh79086**
Issue:
on another pr i face many problem.

### CODE REVIEW

File: src/components/animated/_vibrate.less [hunk 1]
```diff
@@ -0,0 +1,24 @@
+.vibrate-btn&:hover {
```
Comment (shahednasser) [inline src/components/animated/_vibrate.less:1]: remove the &

**Review by shahednasser — CHANGES_REQUESTED** [review]
All looks good just one remark below

File: src/components/animated/_vibrate.less (hunk 1 above)
Comment (adrsh79086) [reply src/components/animated/_vibrate.less:1]: okey

**Comment by shahednasser** [comment]
Great work thank you!

@all-contributors please add @adrsh79086 for code and idea

**Pull request merged**